runtime/libpgmath/lib/common/fpcvt.c

```diff
@@ -9,7 +9,6 @@
 #include <string.h>
 #include <ctype.h>
 #include <errno.h>
-#include "pgstdinit.h"
```

Comment:
Should this file be removed if it is not used anywhere else?

Reply:
Yes, please remove the file. We have already done so in our source tree.

Reply:
Done.